src/components/common/SubmitBtn/_SubmitBtn.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../utils/scss/constants';
+
+.submit-button {
+  display: inline-block;
+  font-size: $font-size-Xsmall;
+  background-color: $color-redish-four;
+  text-decoration: none;
+  color: $color-greyish-five;
+  cursor: pointer;
+  padding: 4px 8px;
+  font-weight: 100;
```

Comment:
osobiście dla czytelności kodu css stosuję taką kolejność/schemat:
position
top/left/...
display

margin
padding
width/height

color
bg-col
border

font...

transition

....

ułatwia mi to szukanie jakichś właściwości, bo wiem gdzie powinny byc.

Reply:
Tutaj przyznam że specjalistą nie jestem - ten css jest zwyczajną kopią od Darka z jego buttona - bo wizualnie to miał być ten sam przycisk tylko potrzebowałem innego typu niż ten przygotowany przez niego.